aries_cloudagent/wallet/routes.py

```diff
@@ -71,6 +71,14 @@ class SetTagPolicyRequestSchema(Schema):
         description="List of attributes to set taggable for credential search",
     )
 
+class SetDidCreateRequestSchema(Schema):
+    """Request did seed."""
+
+    seed = fields.Str(
+        required=False,
+        description="seed for creating did",
+        example="00000000000000000000000000sample"
+    )
```

Comment:
This should have a regex to check validity

Reply:
I think it's good thinking though; I can plumb this in with appropriate unit tests and submit.

Reply:
Thx! :)